src/mruby.rs

```diff
@@ -533,7 +533,7 @@ pub trait MrubyImpl {
     ///
     /// ```
     /// # #[macro_use] extern crate mrusty;
-    /// use mrusty::*;
+    /// use mrusty::{Mruby, MrubyFile, MrubyImpl, MrubyType};
```

Comment:
Do you think a `prelude`  mod would work well here? I generally find these imports not to be terribly bad, but I don't know about the general user.